### ISSUE TITLE
Explore and Create Screen names should match Home Screen button labels 

### ISSUE THREAD

**Issue by Matthew-Moore240** [phetsims/ratio-and-proportion#102]
Buttons should be labelled "Explore" and "Create" instead of "screen 1" and "Screen 2"

**Comment by zepumph** [comment]
This is a sim-specific issue that will be fixed as part of work over in https://github.com/phetsims/joist/issues/641#issuecomment-659445273. On hold until that is fixed, then we can make sure RaP was fixed.

**Comment by zepumph** [comment]
>Buttons should be labelled "Explore" and "Create" instead of "screen 1" and "Screen 2"

They are now labelled "Explore Screen" and "Create Screen" as specified over in #641. Please let me know if that is not acceptable.

**Comment by terracoda** [comment]
These changes were made ready for interviews last week. Closing.
